### PR TITLE
action: use snapshoty simple

### PR DESCRIPTION
**Pull request by v1v**
### What

Simplify the daily snapshots publishing by using the composite action.

It runs on push to `main`
Secrets are not available for PRs coming from forked repos.

### Why

Delegate the implementation details to the composite action.
Avoid errors when running PRs from forked repo

### Follow ups

Support for PRs but for now we only care about snapshots from the main branch.
Change branch to current in the action version

### Issues
Depends on https://github.com/elastic/apm-pipeline-library/pull/2016

### CODE REVIEW

File: .github/workflows/snapshoty.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+permissions:
+  contents: read
+
+jobs:
+  process:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Publish snaphosts
+        uses: elastic/apm-pipeline-library/.github/actions/snapshoty-simple@feature/use-snapshoty-with-docker
```
Comment (v1v) [inline .github/workflows/snapshoty.yml:16]: This will be changed in a follow up as I'd like to test this once it's merged

**Comment by v1v** [comment]
@amannocci , I might need your 👀 as you worked on this in the past 🙏

**Review by LikeTheSalad — APPROVED** [review]

**Review by cachedout — APPROVED** [review]
Much cleaner. 💯

File: .github/workflows/snapshoty.yml (hunk 1 above)
Comment (v1v) [reply .github/workflows/snapshoty.yml:16]:
```suggestion
        uses: elastic/apm-pipeline-library/.github/actions/snapshoty-simple@current
```

**Review by reakaleek — APPROVED** [review]

**Review by pazone — APPROVED** [review]

**Pull request merged**